stix_shifter_modules/lang_en.json

```diff
@@ -4,42 +4,48 @@
             "description": "Datasource type"
         },
         "name": {
-            "label": "Connector name",
-            "placeholder": "QRadar Datasource",
-            "description": "Assign a name to uniquely identify the data source connection"
+            "label": "Data Source Name",
+            "placeholder": "Add a data source name"
         },
         "description": {
-            "label": "Connector Description",
-            "placeholder": "QRadar Datasource Description",
-            "description": "Write a description to indicate the purpose of the data source connection"
+            "label": "Data Source Description",
+            "placeholder": "Add a data source description"
         },
-        "concurrent": {
-            "label": "Concurrent Search Limit",
-            "description": "The number of simultaneous connections that can be made between Cloud Pak for Security and the data source. The default limit for the number of connections is 4."
-        },
-        "result_limit": {
-            "label": "Result Size Limit",
-            "description": "The maximum number of entries or objects that are returned by search query.The default result size limit is 1000. The value must not be less than 1 and must not be greater than 10,000."
-        },
-        "time_range": {
-            "label": "Query Time Range",
-            "description": "Query Time Range"
-        },
-        "timeout": {
-            "label": "Query Timeout Limit",
-            "description": "The time limit in minutes for how long the query is run on the data source. The default time limit is 1. When the value is set to zero, there is no timeout. If the query takes longer than 1 min, it is likely to indicate a problem."
+        "options": {
+            "result_limit": {
+                "label": "Result Size Limit",
+                "description": "The maximum number of entries or objects that are returned by search query."
+            },
+            "time_range": {
+                "label": "Query Time Range",
+                "description": "Time range for the search"
```

Comment:
I wonder if you should add in minutes to this description. Also, should be make it clear it's a time range going back from the current time?